frontend/dashboard/src/features/commands/ui/edit-modal.vue

```diff
@@ -492,6 +492,14 @@ const showCategoryModal = ref(false)
 								autosize
 							/>
 						</NFormItem>
+						<NFormItem
+							:label="t('commands.modal.expiration.label')" path="expiredIn"
+						>
+							<NInputNumber
+								v-model:value="formValue.expiredIn"
```

Comment:
We need to somehow implement `dateTime` picker and isntead save timestamp in database in `expiresAt`.